src/main.js

```diff
@@ -0,0 +1,161 @@
+
+toggleMenuc.addEventListener('click', () => {
+    navMenuc.classList.toggle('show_menu_4')
+})
+closeMenuc.addEventListener('click', () => {
+    navMenuc.classList.remove('show_menu_4')
+})
+
+// link-5
+const navMenud = document.getElementById('nav-menu-5')
+const toggleMenud = document.getElementById('toggle-menu-5')
+const closeMenud = document.getElementById('close-menu-5')
+
+toggleMenud.addEventListener('click', () => {
+    navMenud.classList.toggle('show_menu_5')
+})
+closeMenud.addEventListener('click', () => {
+    navMenud.classList.remove('show_menu_5')
+})
+
+// link-6
+const navMenue = document.getElementById('nav-menu-6')
+const toggleMenue = document.getElementById('toggle-menu-6')
+const closeMenue = document.getElementById('close-menu-6')
+
+toggleMenue.addEventListener('click', () => {
+    navMenue.classList.toggle('show_menu_6')
+})
+closeMenue.addEventListener('click', () => {
+    navMenue.classList.remove('show_menu_6')
+})
+
+// link-7
+const navMenuf = document.getElementById('nav-menu-7')
+const toggleMenuf = document.getElementById('toggle-menu-7')
+const closeMenuf = document.getElementById('close-menu-7')
+
+toggleMenuf.addEventListener('click', () => {
+    navMenuf.classList.toggle('show_menu_7')
+})
+closeMenuf.addEventListener('click', () => {
+    navMenuf.classList.remove('show_menu_7')
+})
+
+// link-8
+const navMenug = document.getElementById('nav-menu-8')
+const toggleMenug = document.getElementById('toggle-menu-8')
+const closeMenug = document.getElementById('close-menu-8')
+
+toggleMenug.addEventListener('click', () => {
+    navMenug.classList.toggle('show_menu_8')
+})
+closeMenug.addEventListener('click', () => {
+    navMenug.classList.remove('show_menu_8')
+})
+
+// link-9
+const navMenuh = document.getElementById('nav-menu-9')
+const toggleMenuh = document.getElementById('toggle-menu-9')
+const closeMenuh = document.getElementById('close-menu-9')
+
+toggleMenuh.addEventListener('click', () => {
+    navMenuh.classList.toggle('show_menu_9')
+})
+closeMenuh.addEventListener('click', () => {
+    navMenuh.classList.remove('show_menu_9')
+})
+
+// link-10
+const navMenuj = document.getElementById('nav-menu-10')
+const toggleMenuj = document.getElementById('toggle-menu-10')
+const closeMenuj = document.getElementById('close-menu-10')
+
+toggleMenuj.addEventListener('click', () => {
+    navMenuj.classList.toggle('show_menu_10')
+})
+closeMenuj.addEventListener('click', () => {
+    navMenuj.classList.remove('show_menu_10')
+})
+
+// link-11
+const navMenui = document.getElementById('nav-menu-11')
+const toggleMenui = document.getElementById('toggle-menu-11')
+const closeMenui = document.getElementById('close-menu-11')
+
+toggleMenui.addEventListener('click', () => {
+    navMenui.classList.toggle('show_menu_11')
+})
+closeMenui.addEventListener('click', () => {
+    navMenui.classList.remove('show_menu_11')
+})
+
+// মোবাইল ভার্সন ন্যাভিগেশন বার শেষ এখানে
```

Comment:
- Replace comment in English
- There should be a blank line at the end of the file